benchmark/kernel/gcn-ogbn.py

```diff
@@ -0,0 +1,203 @@
+import argparse
+import time
+
+import torch
+import torch.nn.functional as F
+from ogb.nodeproppred import Evaluator, PygNodePropPredDataset
+from torch.profiler import ProfilerActivity, profile
+
+import torch_geometric.transforms as T
+from torch_geometric.nn import GCNConv, SAGEConv
+from torch_geometric.profile import rename_profile_file, trace_handler
+
+
+class GCN(torch.nn.Module):
```

Comment:
Is it possible to merge the `GCN` and `SAGE` into one class since they share the same structure and forward passes.

Reply:
Yes. I have merged them into one, by adding one more argument `use_sage`.

Reply:
Thanks. Looks good!